### ISSUE TITLE
Chore -> Add CI Pipeline -> Github Actions

### ISSUE THREAD

**Issue by ekas** [jslovers/jslovers-official-website#3]

**Comment by Neha** [comment]
I can take this. What we are looking to integrate in the pipeline? @ekas

**Comment by ekas** [comment]
> I can take this. What we are looking to integrate in the pipeline? @ekas

Yes, go on. For now I am looking for adding ESlint and Prettier stages and later on more stages could be added.

**Comment by akshat99812** [comment]
Hey, I could do it.

**Comment by akshat99812** [comment]
The linter gives me these errors how do i fix them
<img width="1436" alt="Screenshot 2024-06-18 at 10 30 21 AM" src="https://github.com/jslovers/jslovers-official-website/assets/138353837/7b1698a0-6b07-4bc1-8d6c-07f64694938b">
